src/interfaces/records/handlers/records-read.ts

```diff
@@ -53,13 +53,13 @@ export class RecordsReadHandler implements MethodHandler {
 
     const newestRecordsWrite = newestExistingMessage as RecordsWriteMessage;
     try {
-      await recordsRead.authorize(tenant, await RecordsWrite.parse(newestRecordsWrite), this.messageStore);
+      await recordsRead.authorize(tenant, await RecordsWrite.parse(newestRecordsWrite), this.storageController.MessageStore);
```

Comment:
hmmm moving everything into `StorageController` to then still have to expose `MessageStore` is unfortunate :(

perhaps we should do something like this?
```
authorizeRecordsRead(tenant: string, recordsRead: RecordsRead, newsetRecordsWrite: RecordsWrite) {
  return recordsRead.authorize(tenant, newestRecordsWrite, this.messageStore);
}
```
this way we don't expose `this.messageStore` and keep it kinda internal :)

Reply:
Addressed in https://github.com/TBD54566975/dwn-sdk-js/pull/338/commits/931ab6bd4e5079229f6b9663de5a442eedfe4983